README.md

```diff
@@ -626,12 +626,23 @@ In such a case, this double event firing can be disabled with the `noLoadEventRe
 
 When loading modules that you know will only use baseline modules features, it is possible to set a rule to explicitly opt-out modules from rewriting. This improves performance because those modules then do not need to be processed or transformed at all, so that only local application code is handled and not library code.
 
-This can be configured by providing a URL regular expression for the `skip` option:
+The `skip` option supports a string regular expression, function or array of exact module URLs to check:
```

Comment:
Is this documentation up to date? It seems like the PR is adding support for arrays, but this line implies functions are also supported.
```suggestion
The `skip` option supports a string regular expression or array of exact module URLs to check:
```

Reply:
Thanks for spotting that.